scripts/_helpers.py

```diff
@@ -391,6 +391,38 @@ def _get_country(target, **keys):
         return np.nan
 
 
+def _getContinent(code):
```

Comment:
**Could be addressed**

* Most function names do not use CamelCase naming convention, but follow PEP8 recommendations.
* Leading `_` underscores by convention indicate internal functions. This is a helper function to be used outside the script, i.e. the leading `_` should be removed.

For more info, see: https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

```suggestion
def get_continent(code):
```

Reply:
Thanks, I removed the underscore in the helpers